platforms/common/glfwApp.cpp

```diff
@@ -64,16 +66,20 @@ void create(std::shared_ptr<Platform> p, std::string f, int w, int h) {
         return;
     }
 
-#ifndef MAPZEN_API_KEY
-    LOG("Environment variable MAPZEN_API_KEY not set. Kindly set this environment variable and relaunch.");
-    exit(1);
-#endif
+    char* mapzenApiKeyEnvVar = getenv("MAPZEN_API_KEY");
+    if (mapzenApiKeyEnvVar && strlen(mapzenApiKeyEnvVar) > 0) {
+        mapzenApiKey = mapzenApiKeyEnvVar;
+    } else {
+        LOGW("No API key found!\n\nMapzen data sources require an API key. "
+             "Sign up for a free key at http://mapzen.com/developers and then set it from the command line with: "
+             "\n\n\texport MAPZEN_API_KEY=YOUR_KEY_HERE\n");
```

Comment:
Good point, that could be confusing. Maybe it's easier to just link to https://github.com/tangrams/tangram-es/blob/master/platforms/osx/README.md